test/recipes/controls/aws_parallelcluster_config/network_interfaces_spec.rb

```diff
@@ -42,7 +63,7 @@
         its('group') { should eq 'root' }
         its('content') { should match /^ip route del/ }
       end
-    else
+    elsif os_properties.amazon_family? || os_properties.centos?
```

Comment:
Question: is the change to make the branch explicit?

Reply:
This test doesn't work on RHEL8. I thought of removing it at all, then I opted for excluding just RHEL8.